crates/c-api/include/wasmtime/linker.h

```diff
@@ -66,6 +66,16 @@ WASM_API_EXTERN void wasmtime_linker_delete(wasmtime_linker_t *linker);
 WASM_API_EXTERN void wasmtime_linker_allow_shadowing(wasmtime_linker_t *linker,
                                                      bool allow_shadowing);
 
+/**
+ * \brief Configures whether the given Linker will allow unknown exports from
+ * command modules.
+ *
+ * By default this setting is `false`.
+ */
+WASM_API_EXTERN void
+wasmtime_linker_allow_unknown_exports(wasmtime_linker_t *linker,
+                                      bool allow_unknown_exports);
```

Comment:
I don't see a corresponding function definition in `crates/c-api`, I think it is missing?

Reply:
I think this is a stray declaration from some of our downstream experiments. It has been needed to capture the C shadow stack pointer, that clang generates. It must be kept in sync with the stack switching done by the engine. To be clear, this is just due to the fact that clang doesn't know about stack switching.

Reply:
FWIW, exposing these things the C API would be good in general, so if you want to do it here or in a new PR, you're more than welcome to!

Reply:
Yes sure! I am leaning towards adding this in a follow-up PR focused on the C API. What do you think @frank-emrich ?